packages/react-checkbox/src/Checkbox.LabelBefore.stories.tsx

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
+import { Checkbox } from './index';
+
+export const LabelBefore = () => <Checkbox labelPosition="before" label="Label before" />;
+LabelBefore.parameters = {
+  docs: {
+    description: {
+      story: 'The label can be placed before the checkbox.',
```

Comment:
I wonder if we should have a "please don't do this" type warning here too?

Reply:
There wasn't any such guidance in the checkbox figma 🤷‍♂️